tests/functional/vault/test_misc.py

```diff
@@ -2,12 +2,97 @@
 import brownie
 import pytest
 
+MAX_UINT256 = 2 ** 256 - 1
+MAX_BPS = 10000
+
 
 @pytest.fixture
 def other_token(gov, Token):
     yield gov.deploy(Token, 18)
 
 
+@pytest.fixture
+def token_false_return(gov, TokenFalseReturn):
+    yield gov.deploy(TokenFalseReturn, 18)
+
+
+def vault(gov, management, token, Vault):
```

Comment:
Missing `fixture`?

Reply:
@snorkr4 do you know why this is by any chance? I believe this came from your PR originally. I noticed if I add the fixture the test fails

Reply:
I've made `vault` a fixture and fixed the broken unit tests